hw/ip/kmac/dv/cov/kmac_masked_cov_excl.el

```diff
@@ -1,16 +1,69 @@
-Branch 11 "4188171746" "(!rst_ni)" (4) "(!rst_ni) 0,0,1,-,1,-"
+CHECKSUM: "1221190501 1235822410"
+INSTANCE: tb.dut.u_sha3
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 1 "1305100149" "(process_i && ((!processing))) 1 -1" (2 "10")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 2 "1249907635" "(process_i || run_i || done_i) 1 -1" (2 "001")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 2 "1249907635" "(process_i || run_i || done_i) 1 -1" (3 "010")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 2 "1249907635" "(process_i || run_i || done_i) 1 -1" (4 "100")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 3 "277767244" "(start_i || run_i || done_i || (process_i && processing)) 1 -1" (2 "0001")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 3 "277767244" "(start_i || run_i || done_i || (process_i && processing)) 1 -1" (3 "0010")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 3 "277767244" "(start_i || run_i || done_i || (process_i && processing)) 1 -1" (4 "0100")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 3 "277767244" "(start_i || run_i || done_i || (process_i && processing)) 1 -1" (5 "1000")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 4 "3045477550" "(process_i && processing) 1 -1" (3 "11")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 5 "3212037712" "(start_i || process_i) 1 -1" (2 "01")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 5 "3212037712" "(start_i || process_i) 1 -1" (3 "10")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 6 "1787857586" "(start_i || process_i || run_i || done_i) 1 -1" (2 "0001")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 6 "1787857586" "(start_i || process_i || run_i || done_i) 1 -1" (3 "0010")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 6 "1787857586" "(start_i || process_i || run_i || done_i) 1 -1" (4 "0100")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 6 "1787857586" "(start_i || process_i || run_i || done_i) 1 -1" (5 "1000")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 7 "954210925" "(start_i || process_i || run_i || done_i) 1 -1" (2 "0001")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 7 "954210925" "(start_i || process_i || run_i || done_i) 1 -1" (3 "0010")
+ANNOTATION: "VC_COV_UNR: V2S item- reachable by probing internal state."
+Condition 7 "954210925" "(start_i || process_i || run_i || done_i) 1 -1" (4 "0100")
```

Comment:
These are sha3 exclusions that we can consider probing and forcing them to error state in V2S.